SU-HSDO/suhumsci
# feat(STN-376): add indicators to unpublished pages and menu links

# READY FOR REVIEW

## Summary
Add styles for unpublished pages inculding:
- A banner across the top of any unpublished page
- A highlight color for unpublished pages in the main nav (both 1st and 2nd level)
- A highlight color for unpublished pages in the secondary nav

![image](https://user-images.githubusercontent.com/1256329/82265326-b7d8a880-9934-11ea-80a3-ce166a6b3519.png)

![image](https://user-images.githubusercontent.com/1256329/82265448-fcfcda80-9934-11ea-81e6-d8c85de9dd5a.png)


## Need Review By (Date)
5/25

## Urgency
low

## Steps to Test
1. Pull down this branch 
2. Browse to each of the following types of pages and unpublish them:
    - A page in the 1st level of the main nav
    - A page in the 2nd level of the main nav
    - A page in the top level of the secondary nav
    - A page in a nested level of the secondary nav
3. For each of these pages, confirm that the following:
    1. The page shows the "Unpublished" banner across the top of the page (see screenshot).
    2. The nav link to the page displays the "unpublished background-color" (at mobile and desktop sizes).
    3. The nav link to the page displays the darkened "unpublished" color on hover  (at mobile and desktop sizes).
    4. That the nav link works as described (in steps ii and iii) when it is the active page (the page you are currently on).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
